src/results/proforma.jl

```diff
@@ -120,19 +108,95 @@ function proforma_results(p::REoptInputs, d::Dict)
             year_one_fuel_cost_bau = d["Generator"]["year_one_fuel_cost_before_tax_bau"]
         end
         if !third_party
-            annual_om = -1 * (fixed_and_var_om + d["Generator"]["year_one_fuel_cost_before_tax"])
```

Comment:
@Bill-Becker  I don't think we need a third_party if statement here. Just as you've done for the other techs (ExistingBoiler, Boiler, CHP), I think we can just get the `om_series`, and `fuel_cost_series` (opt and bau) here. Then, they will appropriately be added to the owner vs. offtaker cashflows in the third party if statements down below. 

As it is now, the Generator fuel costs get zeroed out if it's a 3rd party analysis and therefore don't get included in e.g. `r["offtaker_annual_free_cashflows"]`.

Reply:
Agreed, done.